server/services/prison/PrisonRegisterService.ts

```diff
@@ -38,6 +38,15 @@ export default class PrisonRegisterService {
       : Promise.reject(new Error(`PrisonAddress for prison ${prisonId} not found`))
   }
 
+  /**
+   * Returns the name of the prison (`premise` field from the PrisonAddress) for the specified prisonId
+   * or simply the prisonId if the prison cannot be found by it's ID
+   */
+  getPrisonNameOrId(prisonId: string): string {
```

Comment:
I added this to the service class, and deliberately decided to make it return the passed in prisonID if the corresponding prison was not found (rather than an async method that throws).
If it were an async method that throws then it means the calling code (in my case the new Error Response Mapper class) would need to mess about handling it .... and it just looked messy and horrible